### PR TITLE
Translation update for 'react-aria/dnd' and 'react-spectrum/picker'

### PR DESCRIPTION
**Pull request by rgeraghty**
1. Translation update for react-aria/dnd

2. Translation fix for react-spectrum/picker, for Spanish (WAB-27833)

### CODE REVIEW

File: packages/@react-spectrum/picker/intl/es-ES.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "loading": "Cargando…",
-  "placeholder": "Seleccione una opción…"
+  "placeholder": "Selecciona una de estas opciones..."
 }
```
Comment (rgeraghty) [inline packages/@react-spectrum/picker/intl/es-ES.json:4]: Linguistic bug fix verified by translators https://jira.corp.adobe.com/browse/WAB-27833

**Review by dannify — APPROVED** [review]

**Comment by LFDanLu** [comment]
GET_BUILD

**Review by LFDanLu — APPROVED** [review]
LGTM, confirmed the translations for dnd descriptions are coming though dnd (long press and keyboard drag). Confirmed that the Picker placeholder has been updated as well for Spanish

**Review by lydiashortridge36 — APPROVED** [review]
English

**Pull request merged**